README.rst

```diff
@@ -134,9 +134,9 @@ GitHub issue.
 Installing DALI
 ---------------
 
-To install the latest DALI release for the latest CUDA version (11.x)::
+To install the latest DALI release for the latest CUDA version (12.x)::
```

Comment:
Should we update this now?

Reply:
Good point, it may be miesleading for over 1.5 month.

Reply:
Reverted